test/stdlib/Dispatch.swift

```diff
@@ -85,3 +85,20 @@ DispatchAPI.test("dispatch_data_t deallocator") {
 		expectEqual(1, t)
 	}
 }
+
+DispatchAPI.test("DispatchTime comparisons") {
+    do {
+        let now = DispatchTime.now()
+        expectLT(now, .distantFuture)
```

Comment:
Oops I didn't mean to leave this line in here. This was testing something else. I'm going to get rid of it.